src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AuditController.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.pip.account.management.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pip.account.management.authentication.roles.IsAdmin;
+import uk.gov.hmcts.reform.pip.account.management.model.AuditLog;
+import uk.gov.hmcts.reform.pip.account.management.model.AuditLogDto;
+import uk.gov.hmcts.reform.pip.account.management.service.AuditService;
+
+import javax.validation.Valid;
+
+@RestController
+@Tag(name = "Account Management - API for managing audit logs")
+@IsAdmin
+@RequestMapping("/audit")
+public class AuditController {
+
+    private final AuditService auditService;
+
+    private static final String OK_ERROR_CODE = "200";
+
+    @Autowired
+    public AuditController(AuditService auditService) {
+        this.auditService = auditService;
+    }
+
+    @ApiResponse(responseCode = OK_ERROR_CODE, description = "All audit logs returned as a page.")
```

Comment:
Also need the unauthorised Api responses added on these

Reply:
Done